testing/upload_validation.ts

```diff
@@ -2146,6 +2147,19 @@ ${endpointKey ? 'endpointKey is set' : `requiresEndpointUrl is ${requiresEndpoin
     tools: z.array(toolSchema),
   });
 
+  const skillEntrypointsSchema = zodCompleteObject<SkillEntrypoints>({
```

Comment:
Ah sorry I should have said `zodCompletelyStrictObject`.

The difference is that "strict" also validates that there are no extraneous keys. I think all but like 2-3 cases in this file should be using strict, there's some bad old usage in there to be cleaned up.